portalnet/src/overlay_service.rs

```diff
@@ -1692,27 +1767,19 @@ where
     }
 
     /// Processes a Content response.
-    fn process_content(
-        &mut self,
-        content: Content,
-        source: Enr,
-        request: FindContent,
-        query_id: Option<QueryId>,
-    ) {
+    fn process_content(&mut self, content: Content, source: Enr, query_id: Option<QueryId>) {
         trace!(
             protocol = %self.protocol,
             response.source = %source.node_id(),
             "Processing Content message",
         );
         match content {
-            Content::ConnectionId(id) => debug!(
-                protocol = %self.protocol,
-                "Skipping processing for content connection ID {}",
-                u16::from_be(id)
-            ),
```

Comment:
Learning moment: this log should have probably been a `warn!` or higher, to indicate that core functionality was not implemented. We would have noticed it much earlier...